src/v2/Apps/Categories/Components/StickyNav.tsx

```diff
@@ -0,0 +1,105 @@
+import {
+  Link,
+  Pill,
+  Spacer,
+  themeProps,
+  Swiper,
+  SwiperCell,
+  SwiperRail,
+  SwiperCellProps,
+  SwiperRailProps,
+  Flex,
+} from "@artsy/palette"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { AppContainer } from "v2/Apps/Components/AppContainer"
+import { useNavBarHeight } from "v2/Components/NavBar/useNavBarHeight"
+import { extractNodes } from "v2/Utils/extractNodes"
+import { useMatchMedia } from "v2/Utils/Hooks/useMatchMedia"
+import { scrollIntoView } from "v2/Utils/scrollHelpers"
+import { StickyNav_geneFamiliesConnection } from "v2/__generated__/StickyNav_geneFamiliesConnection.graphql"
+interface StickyNavProps {
+  geneFamiliesConnection: StickyNav_geneFamiliesConnection
+}
+
+const StickyNav: React.FC<StickyNavProps> = props => {
+  const { geneFamiliesConnection } = props
+  const geneFamilies = extractNodes(geneFamiliesConnection)
+  const { mobile, desktop } = useNavBarHeight()
+  const isMobile = useMatchMedia(themeProps.mediaQueries.xs)
+  const navBarHeight = isMobile ? mobile : desktop
+  const stickyNavHeight = 50
+  const offset = navBarHeight + stickyNavHeight + 20
+
+  const handleClick = e => {
+    e.preventDefault()
+    const id = e.currentTarget.hash
+
+    scrollIntoView({ selector: id, offset, behavior: "smooth" })
+  }
+
+  return (
+    <>
+      <Spacer pb={1} />
```

Comment:
Avoid external margins in components. The calling component should be where these are.

Reply:
(Agreed; this is related to my other comments)

Reply:
Thanks, that's a good best practice for me to keep in mind! It's one of those things I've always wondered (where spacing should live in general) about but never quite nailed down